apps/protocol-frontend/src/components/HomeShell.tsx

```diff
@@ -115,11 +124,12 @@ const HomeShell = () => {
                   >
                     {userDataByAddress?.name}
                   </Text>
-                  . We'll reach out as soon as we open up more spots!
+                  . <b>We'll reach out as soon as we open up more spots!</b>
```

Comment:
We may want to use <strong> instead for this for screenreader emphasis-- this is super minor though. Good idea to call this text out